### PR TITLE
Add question to ask for view encapsulation strategy

### PR DESCRIPTION
**Pull request by larisaVasile**

### CODE REVIEW

File: src/ng-generate/table/index.ts [hunk 1]
```diff
@@ -37,6 +37,7 @@ import {TsComponentGenerator} from './generators/ts-component.generator';
 import {addModuleImportToModule} from '@angular/cdk/schematics';
 import ora from 'ora';
 import {WIZARD_CONFIG_FILE} from '../table-prompter/index';
+import {ViewEncapsulation} from '@schematics/angular/component/schema';
```
Comment (ashabatin) [inline src/ng-generate/table/index.ts:40]: Looks like it is unnecessary import here

**Review by ashabatin — APPROVED** [review]

**Review by sha4be — APPROVED** [review]

File: src/ng-generate/table/index.ts (hunk 1 above)
Comment (waltersve) [reply src/ng-generate/table/index.ts:40]: yep

**Review by pavel-s-epam — APPROVED** [review]

File: src/ng-generate/table/index.ts (hunk 1 above)
Comment (larisaVasile) [reply src/ng-generate/table/index.ts]: done

**Comment by michelu89** [comment]
Please update documentation part.

**Review by michelu89 — APPROVED** [review]

**Pull request merged**